### PR TITLE
Convert a few llvm tests

### PR DESCRIPTION
**Pull request by jeremy-rifkin**

### CODE REVIEW

File: test/llvm-ir-parser-tests.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ describe('llvm-ir parseMetaNode', function () {
 
     before(() => {
         const fakeProps = new properties.CompilerProps(languages, properties.fakeProps({}));
-        compilerProps = fakeProps.get.bind(fakeProps, 'c++');
+        compilerProps = (fakeProps.get as any).bind(fakeProps, 'c++');
```
Comment (jeremy-rifkin) [inline test/llvm-ir-parser-tests.ts:42]: There's probably a better way to do these

**Review by mattgodbolt — COMMENTED** [review]
Looks like it needs an `npm run lint` to re-sort the imports

File: test/llvm-ir-parser-tests.ts (hunk 1 above)
Comment (mattgodbolt) [reply test/llvm-ir-parser-tests.ts:42]: yeah but honestly right now let's get something checked in!

**Review by mattgodbolt — APPROVED** [review]
ship ittttt (once the imports are the right way, if I'm not wrong)

**Comment by jeremy-rifkin** [comment]
> Looks like it needs an `npm run lint` to re-sort the imports

Oh thanks for the catch (and changing that to an error)

**Pull request merged**